### PR TITLE
[CHA-187] Added chat template pages (features/protocol)

### PR DESCRIPTION
**Pull request by splindsay-92**
Added placeholders for new chat features and protocols.
[CHA-187](https://ably.atlassian.net/browse/CHA-187)

[CHA-187]: https://ably.atlassian.net/browse/CHA-187?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: COPYRIGHT [hunk 1]
```diff
@@ -1 +1 @@
-Copyright 2015-2022 Ably Real-time Ltd (ably.com)
+Copyright 2015-2024 Ably Real-time Ltd (ably.com)
```
Comment (splindsay-92) [inline COPYRIGHT:1]: Just a curiosity, I spotted this was 2 years old now, do we need to keep it updated? 👀

**Comment by ttypic** [comment]
@splindsay-92 @AndyTWF do we need separate protocol version for Chat SDK? I have concerns, that it will add additional level of complexity.

**Comment by AndyTWF** [comment]
@ttypic Our aim is to keep chat versioned separately to the rest of the Ably API because we didn't want to tie it in to the realtime protocol. Doing so would mean that we'd technically have to bump the protocol version used by all the channels SDKs whenever we deliver new features to chat.

So we'll have a versioning scheme for both the backend endpoints, as well as the SDK itself. The reasons for this are discussed in CHARFC-007 on Confluence :)

**Review by ttypic — APPROVED** [review]
LGTM

**Pull request merged**